### ISSUE TITLE
Feature: Setup Links and Resources page

### ISSUE THREAD

**Issue by michaeldev6** [michaeldev6/dev-a11y#13]
- page should be a bunch of links to useful resources on top of the site itself. Like the actualy WCAG page, where we got ideas and solutions for certain features
- a description should go along with each link
- Some quick suggestions: WCAG, WebAim, AdrianRoselli.com, A11yTO, https://developer.mozilla.org/en-US/docs/Web/HTML/Element
- axe guidelineshttps://dequeuniversity.com/rules/axe/
-https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190814/examples/
